driver-reactive-streams/build.gradle

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'de.fuerstenau.buildconfig' version '1.1.8'
```

Comment:
Todo: unite the buildconfig configuration from the main project

Reply:
Done.  I just removed the buildconfig entirely, so client metadata for a reactive MongoClient now looks just like the callback-based MongoClient did in 3.x:

`driver: { name: "mongo-java-driver|async", version: "4.0.0" }`

which is what we want since we know that for 4.0 reactive _is_ the async driver